packages/amplify-migration-tests/src/__tests__/update_tests/storage_migration_update.test.ts

```diff
@@ -40,18 +41,15 @@ describe('amplify add/update storage(DDB)', () => {
     await addAuthWithDefault(projRoot);
     await addSimpleDDB(projRoot, {});
     await addDDBWithTrigger(projRoot, {});
-    await amplifyPushAuth(projRoot);
+    await amplifyPushAuthLegacy(projRoot);
     // update and push with codebase
     await updateDDBWithTriggerMigration(projRoot, { testingWithLatestCodebase: true });
     await amplifyPushAuth(projRoot, true);
 
     const meta = getProjectMeta(projRoot);
-    const {
-      Name: table1Name,
-      Arn: table1Arn,
-      Region: table1Region,
-      StreamArn: table1StreamArn,
-    } = Object.keys(meta.storage).map(key => meta.storage[key])[0].output;
+    const { Name: table1Name, Arn: table1Arn, Region: table1Region, StreamArn: table1StreamArn } = Object.keys(meta.storage).map(
```

Comment:
nit: if this was autoformatted it's fine to leave it, but I liked the previous formatting better

Reply:
yes, this is auto-formatted